tests/includes/ApplicationBaseTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Matthew
+ * Date: 2/22/16
+ * Time: 13:53
+ */
+
+namespace Waca\Tests;
+
+class ApplicationBaseTest extends \PHPUnit_Framework_TestCase
+{
+	private $si;
+    private $ab;
+
+	public function setUp() {
+		$this->si = new \Waca\siteConfiguration();
```

Comment:
`SiteConfiguration` has a capital S (I know PHP doesn't care, but the autoloader does on case-sensitive file systems, which could cause class load issues on some systems).